app/src/components/Card/CardPreview/index.tsx

```diff
@@ -40,56 +39,97 @@ function CardPreview({ card, onClick }: ICardPreviewProps) {
+        </div>
+        <div className="flex-grow overflow-hidden">
+          {!isEmpty(tasks) ? (
+            <div className="flex flex-col gap-3">
+              <div className="col-span-3 overflow-hidden">
+                {tasks.slice(0, 3).map((task) => (
+                  <div key={'prevtask-' + task.id} className="h-[46px] py-1 justify-start items-start gap-2 inline-flex">
+                    <div className="w-4 h-4 justify-center items-center flex">
+                      <div className="w-4 h-4 pl-[1.60px] pr-[1.23px] pt-[1.60px] pb-[1.30px] justify-center items-center inline-flex">
+                        <div className="w-[13.17px] h-[13.10px] relative">
+                          {/* Здесь можно вставить содержимое, например, иконку */}
+                        </div>
+                      </div>
+                    </div>
+                    <div className="grow shrink basis-0">
+                      <span className="text-[#46434e] text-sm font-normal font-['Rubik'] leading-[18.90px]">
+                        {task.title}
+                      </span>
+                      <span className="text-[#46434e] text-sm font-semibold font-['Rubik'] leading-[18.90px]">
+                        @MattD
+                      </span>
+                    </div>
                   </div>
                 ))}
-          </div>
-          {tasks?.length > 4 ? (
-            <span className='ml-3 flex items-center h-full underline text-sm'>
-              +{tasks.length - 4}
-            </span>
-          ) : null}
+              </div>
+              {tasks.length > 4 && (
+                <span className='ml-3 flex items-center h-full underline text-sm'>
+                  +{tasks.length - 4}
+                </span>
+              )}
+            </div>
+          ) : (
+            <div className='flex-grow overflow-hidden'>
+              {card.description.map((line, index) => (
+                <div key={'description-' + line + index} className="text-[#46434e] text-sm font-normal font-['Rubik'] leading-tight overflow-hidden" style={{ display: '-webkit-box', WebkitLineClamp: 2, WebkitBoxOrient: 'vertical', overflow: 'hidden', textOverflow: 'ellipsis' }}>
+                  {line}
+                </div>
+              ))}
+            </div>
+          )}
         </div>
-        {/* <div className='grid grid-cols-5 mt-5'> */}
-        <div className='flex mt-5 sm:max-w-[390px] md:max-w-[300px] lg:max-w-[450px] '>
-          {!isEmpty(imageUrls) ? (
-            <div className='flex col-span-2 '>
-              {imageUrls?.map((url, idx) => (
-                <Image key={'image-' + url + '-' + idx} className='w-12' src={url} alt='' />
+        <div className="mt-auto w-full">
+          <div className="h-[56px] w-full pr-2 bg-white rounded-sm justify-start items-center gap-1 inline-flex">
+            <div className="justify-start items-center flex w-full">
+              {(imageUrls ?? []).slice(0, 4).map((url, index) => (
```

Comment:
Actually we split this array on 29 line. Could you check my improvements in 29 line 👆

Reply:
And if we have imageUrls === [] (0 length)  - this means that if there are no images, this block should not be rendered at all.